requirements.txt

```diff
@@ -2,6 +2,7 @@ Authlib==1.0.1
 cryptography==37.0.2
 Django==3.2.13
 django-csp==3.7
+git+https://github.com/cal-itp/eligibility-api@54f93e20ed2adb797b7bcd94a14b16fb3f8c2342#egg=eligibility_api # WIP: pointing at specific commit with types_to_verify fix
```

Comment:
I recommend we point at `main` so we don't have to constantly update this reference here.

Each time the container is rebuilt it will grab the latest `main` from API. So we just have to make sure that it always stays stable.